intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -736,6 +744,17 @@ \section{Transfer Files to/from the \hpc}
 -rw-r--r--   1 user  staff       6 Sep 18 09:37 localfile.txt
 \end{prompt}
 
+The file has been copied from the HPC to your local computer.
+
+It's also possible to copy entire directories (and their contents) with the \lstinline|-r| flag.
+For example, if we want to copy the local directory \lstinline|dataset| to \lstinline|$VSC_SCRATCH|,
+we can use the following command (assuming you've created the \lstinline|scratch| symlink):
+
+\begin{prompt}
+%\shellcmd{scp -r dataset \userid{}@\loginnode{}:scratch}%
+\end{prompt}
+
```

Comment:
Also mention error that you'll run into when not using `-r` when `scp`ing a directory?